src/components/text-editor/__internal__/text-editor.stories.mdx

```diff
@@ -119,17 +119,49 @@ will prevent any input that would cause the content length to exceed it.
   <Story name="with optional character limit">
     {() => {
       const [value, setValue] = useState(EditorState.createEmpty());
-      const [limit, setLimit] = useState(100);
+      const limit = 100;
       const ref = useRef(null);
       return (
-          <div style={{ padding: '4px' }}>
-            <TextEditor
-              onChange={(newValue) => {setValue(newValue)}}
-              value={ value } ref={ ref }
-              onSubmit={ () => {} }
-              labelText='Text Editor Label'
-              characterLimit={ limit }
-            />
+        <div style={{ padding: '4px' }}>
+          <TextEditor
+            onChange={(newValue) => {setValue(newValue)}}
+            value={ value } ref={ ref }
+            onSubmit={ () => {} }
+            labelText='Text Editor Label'
+            characterLimit={ limit }
+          />
+        </div>
+      );
+    }}
+  </Story>
+</Preview>
+
+### With validation
+It is possible apply validation to the `TextEditor` component by passing the desired string message to one of the `error`, 
+`warning` or `info` props. When an `error` message is provided the border styling of the editor is updated to indicate 
+this status.
+
+<Preview>
+  <Story name="with validation">
+    {() => {
+      const [value, setValue] = useState(
+        EditorState.createWithContent(ContentState.createFromText('Add content'))
+      );
+      const [limit, setLimit] = useState(16);
```

Comment:
Is `setLimit` used here, you removed the declaration at line 122?

Reply:
Ah good spot I'll remove it